core/api/src/graphql/admin/schema.graphql

```diff
@@ -443,7 +460,10 @@ interface Wallet {
   accountId: ID!
   balance: SignedAmount!
   id: ID!
+  invoiceByHash(paymentHash: PaymentHash!): LnInvoice!
   pendingIncomingBalance: SignedAmount!
+  transactionByHash(hash: PaymentHash!): Transaction!
```

Comment:
should the type be mandatory `Transaction!`, or should it be optional? what happen when the transaction/invoice doesn't exist?

Reply:
current the type we have in the admin panel is not mandatory (not saying it should one or the other, not sure yet, but I think it should be consistent)